robot/board.py

```diff
@@ -155,6 +155,9 @@ def close(self):
         """
         self.socket.detach()
 
+    def __repr__(self):
+        return "<{}>".format(self.__str__())
```

Comment:
You shouldn't need to call `__str__` explicitly here.

Reply:
fixed in f6b302caa0584e038006c46df4e43ae474f7910e